beacon-chain/forkchoice/doubly-linked-tree/forkchoice.go

```diff
@@ -115,8 +115,27 @@ func (f *ForkChoice) InsertNode(ctx context.Context, state state.BeaconState, ro
 		return errNilBlockHeader
 	}
 	parentRoot := bytesutil.ToBytes32(bh.ParentRoot)
-	var payloadHash [32]byte
-	if state.Version() >= version.Bellatrix {
+	var payloadHash, parentHash [32]byte
+	if state.Version() >= version.EPBS {
+		slot, err := state.LatestFullSlot()
+		if err != nil {
+			return err
+		}
```

Comment:
Spec has `IsParentBlockFull`, I wonder if something like `IsCurrentblockFull` will be nice to have as a prysm helper or even have it in the spec. I don't see how to use it in the spec though, maybe it's just a prysm implementation detail